Importantlinks.md

```diff
@@ -0,0 +1,45 @@
+#RUST LINKS
+------------
+
+####1. Rust Tutorial
```

Comment:
Use Hyperlink 

ex:
~~~~
 [text of the link](link)
~~~~

output: [text of the link](link)

Use bullets

ex:
~~~~
* Resource 1
~~~~

* Resource